cmd/kube-apiserver/app/server.go

```diff
@@ -579,6 +579,7 @@ func BuildPriorityAndFairness(s *options.ServerRunOptions, extclient clientgocli
 		extclient.FlowcontrolV1alpha1(),
 		s.GenericServerRunOptions.MaxRequestsInFlight+s.GenericServerRunOptions.MaxMutatingRequestsInFlight,
 		s.GenericServerRunOptions.RequestTimeout/4,
+		s.SecureServing.BindAddress,
```

Comment:
IMO it makes sense to do the full stringification here and simply pass a string ID to `utilflowcontrol.New` instead of passing a `net.IP`. The kube-apiserver has more information at this stage, so it can potentially use other fallback identifiers (instead of timestamp).

Reply:
The whole identity part of this is just a placeholder until https://github.com/kubernetes/enhancements/issues/1965 is done.

Reply:
can a beta feature depend on an alpha feature? in any case, can we add a TODO here referencing that KEP?

Reply:
I added a comment on the function taking the interim ID.